unforbid.lua

```diff
@@ -22,6 +22,11 @@ local function unforbid_all(include_unreachable, quiet)
                     if not quiet then print(('  unreachable: %s (skipping)'):format(item)) end
                     goto skipitem
                 end
+
+                if ((not options.include_tattered) and item.wear >= 3) then
```

Comment:
The threshold is debatable. Should it be configurable? I can see the player wanting it to match whatever `cleanowned` is set to. What should be the default? We should probably match the default `cleanowned` command in the control panel.